app/src/main/java/com/pajato/android/gamechat/exp/TileClickHandler.java

```diff
@@ -71,7 +71,7 @@ private boolean isPlayingOutOfTurn(final int position) {
         // If so, return false, otherwise true.
         Board board = mModel.getBoard();
         Team team = board.getTeam(position);
-        if (team == Team.NONE) // clicking on an empty cell should be ignored
+        if (team == Team.NONE) // clicking on an empty cell is not playing out of turn
```

Comment:
It would be better to include the note about clicking on an empty cell into the comment above such that it reads naturally, i.e. the comment probably needs to be rewritten entirely such that the comment and the code it covers are consistent.